src/ensembl/src/content/app/browser/zmenu/ZmenuController.tsx

```diff
@@ -104,5 +109,11 @@ const ZmenuController = (props: Props) => {
 
   return <>{zmenuElements}</>;
 };
+const mapDispatchToProps = {
+  changeHighlightedTrackId
+};
 
-export default React.memo(ZmenuController);
+export default connect(
+  undefined,
+  mapDispatchToProps
+)(React.memo(ZmenuController));
```

Comment:
connect already does some props checking to decide whether its child component should update. I think we can safely remove React.memo.

Reply:
That's done